src/FondOfSpryker/Yves/GoogleTagManagerCategoryConnector/Expander/CategoryDataLayerExpander.php

```diff
@@ -0,0 +1,123 @@
+<?php
+
+namespace FondOfSpryker\Yves\GoogleTagManagerCategoryConnector\Expander;
+
+use FondOfSpryker\Shared\GoogleTagManagerCategoryConnector\GoogleTagManagerCategoryConstants as ModuleConstants;
+
+class CategoryDataLayerExpander implements CategoryDataLayerExpanderInterface
```

Comment:
Use this pattern for your expander in this module: https://refactoring.guru/design-patterns/chain-of-responsibility/php/example